### DISCUSSION TITLE
关于actions 的脚本规则如何学习使用

### DISCUSSION THREAD

**Discussion by wangxinleo** [wangxinleo/wechat-public-account-push#21] [Q&A]
具体在https://docs.github.com/cn/actions/using-workflows/events-that-trigger-workflows

自行查看 cron规则。

**Comment by LordonCN** [comment]
![截屏2022-08-28 12 30 12](https://user-images.githubusercontent.com/46736284/187057548-30e9554d-30bc-446a-ae7f-1c1eff3dfbd2.png)

贴一个更直观hhh
